knexfile.js

```diff
@@ -19,6 +19,19 @@ module.exports = {
     }
   },
 
+  test: {
+    client: 'pg',
+    connection: {
+      database: 'voicer',
+      user:     'postgres',
+      password: ''
+    },
+    useNullAsDefault: true,
+    migrations: {
+      directory: './data/migrations'
+    }
+  },
+
```

Comment:
should we import the password as a dotenv?

Reply:
maybe use TEST_PASSWORD

Reply:
It is a local database so a password isn't necessary.